test/bootstrap.php

```diff
@@ -7,7 +7,7 @@
 /*
  * Set error reporting to the level to which Laminas code must comply.
  */
-error_reporting(E_ALL | E_STRICT);
+error_reporting(E_ALL | @E_STRICT);
```

Comment:
Wouldn't `E_ALL` suffice here? `E_STRICT` has been part of `E_ALL` for ages hasn't it? So adding `E_STRICT` anywhere is superfluous I thought…

Reply:
I guess for PHP 8.4 it would, but for the earlier versions we probably need both.

Reply:
I just checked - E_STRICT is included in E_ALL since PHP 7.something and it's definitely included in 8.1+ so it can be reduced just to `error_reporting(E_ALL)`

Reply:
Done